### PR TITLE
Added the Swift Package Manager

### PR DESCRIPTION
**Pull request by OscarGorog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,15 @@ Afterwards, run the following command:
 carthage update
 ```
 
+### Installation via Carthage
```
Comment (bok-) [inline README.md]: Still using the Carthage header

Comment (OscarGorog) [reply README.md]: Fixed in commit: ‘064ed96’

**Comment by bmoliveira** [comment]
Can you please review this to the 1.6.0 I approved all the other PR's and now I don't have much time to implement the things that are conflicting, but as soon as you change this I'll merge it. 

Thank you very much

**Comment by benjohnde** [comment]
Closed in favour of #81!

**Pull request closed without merging**